src/main/java/org/update4j/util/PathUtils.java

```diff
@@ -0,0 +1,60 @@
+package org.update4j.util;
```

Comment:
Since `PropertyManager` will _always_ be loaded before this is any useful, I personally would move the logic into the `PropertyManager` class. We now no longer need to call `init()` all over.

Before you change it, I still want to think about adding a few standardized global system properties (or maybe use some other idea instead of setting system property, as automatically adding them to resolved properties map in `PropertyManager` constructor. So we don't mess around with system properties that might have a different use in different contexts).

I personally prefer dot delimited properties:

```
user.appdata
```
How about adding:

```
update4j.version (from Bootstrap.VERSION)
```


Checkout [Inno Setup Constants](http://www.jrsoftware.org/ishelp/index.php?topic=consts) for more ideas.